modules/plugin/jdbc/jdbc-oracle/src/test/java/org/geotools/data/oracle/OracleDataStoreAPITestSetup.java

```diff
@@ -45,6 +45,8 @@ protected void createLakeTable() throws Exception {
         run("INSERT INTO lake (fid,id,geom,name) VALUES (lake_fid_seq.nextval, 0,"
             + "MDSYS.SDO_GEOMETRY( 2003, 4326, NULL, SDO_ELEM_INFO_ARRAY(1,1003,1), "
             + "SDO_ORDINATE_ARRAY(12,6, 14,8, 16,6, 16,4, 14,4, 12,6)), 'muddy')");
+        
+        run ("ANALYZE INDEX LAKE_GEOM_IDX compute statistics");
```

Comment:
Wondering, is there a minimum Oracle version to use this command?

Reply:
It's available in at least 11.1 where there's a note that the compute option is there for backwards compatiblility https://docs.oracle.com/cd/B28359_01/server.111/b28310/general002.htm#ADMIN11524

Reply:
Roger that, merging